### PR TITLE
machdyne: fix typos; add vanille and lakritz

### PR DESCRIPTION
**Pull request by machdyne**
This PR fixes some typos in the Machdyne platform files and adds support for the [Vanille](https://github.com/machdyne/vanille) and [Lakritz](https://github.com/machdyne/lakritz) FPGA computers.

### CODE REVIEW

File: litex_boards/targets/machdyne_lakritz.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        if with_led_chaser:
+            self.leds = LedChaser(
+                pads         = platform.request_all("user_led"),
+                sys_clk_freq = sys_clk_freq)
+
+# Build --------------------------------------------------------------------------------------------
+
+def main():
+    from litex.soc.integration.soc import LiteXSoCArgumentParser
+    parser = LiteXSoCArgumentParser(description="LiteX SoC on Lakritz")
```
Comment (trabucayre) [inline litex_boards/targets/machdyne_lakritz.py:172]: Please switch to `LiteXArgumentParser`

**Comment by trabucayre** [comment]
Looks good.
Vanille board seems to have a problem with CI because it has no serial interface. It's maybe required to add a valenty USB instance or to disable uart interface.

**Comment by trabucayre** [comment]
Applied with a small [fix](https://github.com/litex-hub/litex-boards/commit/75ef26b8e54b7b0c1c5e2fba63336f0752b96c2b). Thanks @machdyne !

**Comment by machdyne** [comment]
Thanks @trabucayre !

**Pull request merged**